scripts/build-icons.js

```diff
@@ -177,41 +185,45 @@ for (const icon of whitelist) {
 }\n`)
 
     // Generate symbol image
-    continue // It's not ready yet
     run(`mkdir -p ${sfolder}`)
     fs.writeFileSync(`${sfolder}/${name}.svg`,
-`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="-192 -192 2304 2304">
+`
```

Comment:
Is this blank first line intentional? It shows up in each svg file. You can put a `.trimStart()` at the end to eliminate it. I think we want the trailing newline though.